src/types/message.ts

```diff
@@ -280,9 +281,10 @@ export function isBadMessage(message: Message): boolean {
 /**
  * All types of messages sent to and from the GCS.
  */
-export type Message = StartMessage | AddMissionMessage | PauseMessage | ResumeMessage | StopMessage
-| ConnectionAckMessage | UpdateMessage | POIMessage | CompleteMessage | ConnectMessage
-| AcknowledgementMessage | BadMessage;
+
+export type Message = | StartMessage | AddMissionMessage | PauseMessage | ResumeMessage |
```

Comment:
just realized you have `Message = | StartMessage` remove the | at the beginning and put StopMessage
back in this line

```ts
export type Message = Message = StartMessage | AddMissionMessage | PauseMessage | ResumeMessage | StopMessage
```